main/lp/learnpath.class.php

```diff
@@ -777,7 +778,13 @@ public static function add_lp(
         if (empty($expired_on)) {
             $expired_on = null;
         } else {
-            $expired_on = Database::escape_string(api_get_utc_datetime($expired_on));
+            $expired_on = api_get_utc_datetime($expired_on, false, true);
+        }
+
+        if (empty($publicated_on)) {
+            $publicated_on = null;
+        } else {
+            $publicated_on = api_get_utc_datetime($publicated_on, false, true);
```

Comment:
This condition doubles the time difference. This should replace the previous condition.

Fixed here 9205b9c